src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -298,10 +298,10 @@ private async Task PushPackagePath(string packagePath,
                     Strings.NoApiKeyFound,
                     GetSourceDisplayName(source)));
             }
-            bool warnForHttpSources = true;
+            bool logErrorForHttpSources = true;
             foreach (string nupkgToPush in nupkgsToPush)
             {
-                bool packageWasPushed = await PushPackageCore(source, apiKey, nupkgToPush, noServiceEndpoint, skipDuplicate, requestTimeout, warnForHttpSources, allowInsecureConnections, log, token);
+                bool packageWasPushed = await PushPackageCore(source, apiKey, nupkgToPush, noServiceEndpoint, skipDuplicate, requestTimeout, logErrorForHttpSources, allowInsecureConnections, log, token);
```

Comment:
This line is 198 chars long. Per [https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#style-guidelines](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#style-guidelines), we should limit line lengths to about 65 chars. Would you please break down into more lines and do a pass for the entire file for the same?